hyperapp.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module "hyperapp" {
+  export interface VNodeData {
+    [key: string]: any
```

Comment:
@Swizz I don't know how to write this file, so I am only asking to learn. Please make the typings in the best way you know. Size doesn't matter.